src/pages/en/guides/troubleshooting.mdx

```diff
@@ -71,6 +71,31 @@ export default defineConfig({
 
 Alternatively, you can update your CSP to include `script-src: 'unsafe-inline'` to allow inline scripts to run.
 
+### exports is not defined
```

Comment:
I used one of the error messages I've seen before, but I think this issue can manifest in many different kinds of errors, so I'm not really sure if this is the right move.